docs/sources/enterprise/access-control/fine-grained-access-control-references.md

```diff
@@ -11,28 +11,32 @@ The reference information that follows complements conceptual information about
+| `fixed:settings:admin:read`           | `settings:read`                                                                                                                                                                                                                                                              | Read settings                                                                                                                             |
+| `fixed:settings:admin:edit`           | All permissions from `fixed:settings:admin:read` and<br>`settings:write`                                                                                                                                                                                                     | Update settings                                                                                                                           |
+| `fixed:datasource:editor:read`        | `datasources:explore`                                                                                                                                                                                                                                                        | Allows to access the **Explore** tab                                                                                                      |
+| `fixed:datasources:admin`             | `datasources:read`<br>`datasources:create`<br>`datasources:write`<br>`datasources:delete`                                                                                                                                                                                    | Allows to create, read, update, delete data sources.                                                                                      |
+| `fixed:datasources:id:viewer`         | `datasources:id:read`                                                                                                                                                                                                                                                        | Allows to read data sources ID.                                                                                                           |
+| `fixed:datasources:permissions:admin` | `datasources.permissions:create`<br> `datasources.permissions:read`<br> `datasources.permissions:delete`<br>`datasources.permissions:toggle`                                                                                                                                 | Allows to create, read, delete, enable, or disable data source permissions                                                                |
```

Comment:
Modified these

Reply:
I like that we are very explicit about what `fixed:datasource:editor:read` role allows to do. I assume in the future it might have more permissions assigned, however good to be clear about it no.